src/main/java/entity/Collectible.java

```diff
@@ -0,0 +1,32 @@
+package entity;
+
+import entity.Item;
+
+public class Collectible extends Item {
+
+    /**
+     * Creates a new entity.Collectible object.
```

Comment:
we should specify here that this constructor will be used only to create instances of artifact, since the count will always start at 0.